protos/protos/public/modeldb/versioning/VersioningService.proto

```diff
@@ -418,6 +418,45 @@ message RevertRepositoryCommitsRequest {
     }
 }
 
+message FindRepositories {
+    repeated uint64 repo_ids = 1;
+    repeated string repo_names = 2;
```

Comment:
I'm not sure we'd need this field. We can add as long as it doesn't add a lot of complexity to the implementation.